roles/postgres/tasks/main.yml

```
@@ -0,0 +1,50 @@
+- name: Install postgres Python library.
+  apt:
+    name: "python3-psycopg2"
+    state: present
+
+- name: Install postgres packages.
+  apt:
+    name:
+      - postgresql
+      - postgresql-contrib
```

Do we need this package? If yes, we should probably instead install:

```suggestion
      - postgresql-contrib-{{ postgresql_version }}
```